server2/selfsign/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
```

Comment:
Should there be a comment in here saying what this docker container does. also a docker run example, with the required environment variables, eg, $WEBSITE, $RANDOM, etc

Reply:
Done.